controllers/kafkatopic_controller.go

```diff
@@ -47,7 +47,7 @@ func SetupKafkaTopicWithManager(mgr ctrl.Manager) error {
 		Log:    ctrl.Log.WithName("controllers").WithName("KafkaTopic"),
 	}
 
-	c, err := controller.New("kafkatopic", mgr, controller.Options{Reconciler: r})
+	c, err := controller.New("kafkatopic", mgr, controller.Options{Reconciler: r, MaxConcurrentReconciles: 10})
```

Comment:
Can you introduce an argument (https://github.com/banzaicloud/kafka-operator/blob/master/main.go#L88-L97) for kafka-operator through which the value for `MaxConcurrentReconciles` can be configured. If not provided than defaults to 10.

Reply:
👌